docs/fleet/rbac.md

```diff
@@ -50,6 +50,12 @@ For more information on the permissions the roles assign for each resource, see
 The option to grant additional access is only visible if you can grant the user additional access.
 {{< /alert >}}
 
+### Use the mobile app
+
+You can also use the [Viam mobile app](/fleet/#the-viam-mobile-app) to grant or revoke org owner or operator access to users on the go.
```

Comment:
Nit: just because ((grant or revoke) + (owner or operator)) is a little complicated, and the abilities of this mobile UI are probably 1:1 with the existing web UI as far as RBAC config goes, I'd imagine. If they are different though, this would be the place to highlight that.

Reply:
I put that specifically because that's the only thing you can do which is different than the web UI!

Reply:
Ah, if they differ, then you're right to be explicit! Thanks for clarifying!